libs/sql-parser/src/main/java/io/crate/sql/tree/CreateTableAs.java

```diff
@@ -56,7 +58,12 @@ public boolean equals(Object o) {
         }
         CreateTableAs<?> that = (CreateTableAs<?>) o;
         return name.equals(that.name) &&
-               query.equals(that.query);
+            query.equals(that.query) &&
+            ifNotExists == that.ifNotExists;
+    }
```

Comment:
Can you also add the ifNotExists property to the hashCode calculation further below to make sure equals/hashCode are consistent?

Reply:
okay. I will look into this.

Reply:
I believe I did it.